src/components/components/ebay-notice-base/index.marko

```diff
@@ -0,0 +1,62 @@
+import processHtmlAttributes from "../../../common/html-attributes"
+
+static var ignoredAttributes = [
+    "status",
+    "a11yText",
+    "iconClass",
+    "class",
+    "root",
+    "headerRoot",
+    "a11yRoleDescription",
+    "prefixClass",
+    "title",
+    "footer"
+];
+
+$ var status = input.status;
+$ var prefixClass = input.prefixClass;
+<${input.root || "section"}
+    aria-labelledby:scoped="status"
+    class=[prefixClass, input.class]
+    ...processHtmlAttributes(input, ignoredAttributes)>
+    <${input.headerRoot || "div"} class=`${prefixClass}__header` id:scoped="status"
+        aria-roledescription=input.a11yRoleDescription
+    >
+
+        <if(input.icon !== "none")>
+            <if(status === "confirmation" || status === "celebration")>
+                <ebay-icon
+                    name="confirmation-filled"
+                    aria-label=input.a11yText
+                    class=input.iconClass/>
```

Comment:
Not a huge fan of this `iconClass` attribute. Let's remember to address this when we look at refactoring the icon apis.